### PR TITLE
go2rtc 1.8.5

### PR DESCRIPTION
**Pull request by skrashevich**
## Description

version update

The patch is needed to provide compatibility with go1.20
Appropriate PR in the main project repository: https://github.com/AlexxIT/go2rtc/pull/865

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Package-Update-Policy#tests-checks)
- [x] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [ ] Bug fix
- [ ] New Package
- [x] Package update
- [ ] Includes small framework changes
- [ ] This change requires a documentation update (e.g. Wiki)

### CODE REVIEW

**Comment by hgy59** [comment]
@skrashevich we do not want to be a development environment for other packages.
Please wait until the upstream repo has implemented your solution and to not update the spk package with such patches.

**Comment by skrashevich** [comment]
All patches have been removed, as compatibility with go1.20 is no longer required (due to #5981)

File: cross/go2rtc/Makefile [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+PKG_VERS = 1.8.5
 PKG_EXT = tar.gz
 PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://github.com/AlexxIT/go2rtc/archive
 PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 EXTRACT_PATH = $(WORK_DIR)/src/github.com/$(PKG_NAME)
 PKG_DIR = src/github.com/$(PKG_NAME)/$(PKG_NAME)
 
+PATCHES_LEVEL = 1
+
```
Comment (hgy59) [inline cross/go2rtc/Makefile:11]: This is obsolete now
```suggestion
```

**Review by hgy59 — APPROVED** [review]
LGTM

**Pull request merged**